tests/test.py

```diff
@@ -0,0 +1,100 @@
+import glob
+import os
+import re
+import subprocess
+import sys
+import time
+import shutil
+import socket
+
+from avocado import VERSION
+from avocado import Test
+
+class MinishiftISOTest(Test):
+
+    def setUp(self):
+        ''' Test Setup '''
+        self.log.info("################################################################")
+        self.log.info("Avocado version : %s" % VERSION)
+        self.log.info("################################################################")
+
+        self.repo_dir = os.path.dirname(os.path.realpath(__file__)) + "/.."
+        self.scripts_dir = self.repo_dir + "/tests/"
+        self.bin_dir = self.repo_dir + "/build/bin/"
+        self.driver_name = 'kvm'
```

Comment:
@budhrg If your intention is to make this work for other OS types right away, then this is indeed not going to work. Why can you not use the default (at least for now). Why setting the driver explicitly?